Nos/Views/Onboarding/CreateAccountView.swift

```diff
@@ -0,0 +1,105 @@
+import Dependencies
+import SwiftUI
+
+/// The Create Account view in the onboarding.
+struct CreateAccountView: View {
+    @Environment(OnboardingState.self) private var state
+
+    @Dependency(\.crashReporting) private var crashReporting
+    @Dependency(\.currentUser) private var currentUser
+
+    var body: some View {
+        GeometryReader { geometry in
```

Comment:
This works, but you can also use the new `ViewThatFits` if you prefer. There's an example in `DiscoverContentsView`

Reply:
Yeah, I'd rather use that. Thanks!